.travis.yml

```diff
@@ -3,12 +3,13 @@ node_js:
   - "0.10"
   - "0.12"
   - "4"
```

Comment:
I'm operating under the assumption that if 4.x will behave functionally the same for this project. If something changes drastically enough that this breaks, it'll be on a major version... then again, 4.2.x is the LTS release... maybe I just go with that in the future?